src/server/search/search_family.cc

```diff
@@ -30,7 +31,7 @@ using namespace facade;
 
 namespace {
 
-unordered_map<string_view, search::Schema::FieldType> kSchemaTypes = {
+const absl::flat_hash_map<string_view, search::Schema::FieldType> kSchemaTypes = {
```

Comment:
Just FYI, in Google (internally) there's a constexpr map for exactly this use case. I can't find it online, so I imagine it's not open source :(

Reply:
I searched for one as well and couldn't find one! 😢